int8 tag_id
WaypointType type
